src/cpp/src/whisper_pipeline_static.cpp

```diff
@@ -498,6 +502,319 @@ std::shared_ptr<ov::Model> redirect_new_kv_to_output(const std::shared_ptr<ov::M
     return model;
 }
 
+bool find_name(const std::string& to_check, const std::string& name) {
```

Comment:
one-liner wrapped into function? Why just not use `to_check.find(name) == std::string::npos`?

Reply:
Fixed